examples/step-97/doc/intro.dox

```diff
@@ -0,0 +1,938 @@
+  boundary condition is applied to this surface.
+
+The program uses four meshes created with a help of gmsh. Each mesh has a
+different degree of refinement. The degree of refinement in each mesh is
+defined by a number of mesh nodes on the transfinite lines, $r$. The figure
+below illustrates a mesh with $r=5$. The program uses for meshes with the
+amount of nodes on the transfinite lines in the interval $r = [6, 9]$.
+
+Note that this tutorial uses only globally refined meshes. There are no
+non-conforming cells, hanging nodes, and hanging node constrains in this
+tutorial. We will use constrains only for the purpose of enforcing the
+Dirichlet boundary condition.
+
+@htmlonly
+<p align="center">
+  <img src="images/steps/developer/step-97-mesh.svg" alt="The mesh" height="531">
+</p>
+@endhtmlonly
+
+<h3>Overview of the program</h3>
+
+The program runs in a loop. In each iteration of the loop a different parameter
+$r$ (the mesh refinement parameter, i.e., the amount of nodes on transfinite
+lines) is assumed. Each iteration consists of four stages:
+
+- The current vector potential, $\vec{T}$, is computed given the closed-form
+  analytical expression for the free-current density, $\vec{J}_f$, see above.
+  No error norms are computed. The mesh is loaded at this stage. This mesh is
+  simply reused at all other stages.
+- The current vector potential computed at the preceding stage, $\vec{T}$, is
+  converted numerically back into free-current density, $\vec{J}_f$, for
+  verification. The $L^2$ error norm is computed by comparing the numerical
+  result with the closed-form analytical expression. The $L^2$ error norm is
+  saved in a convergence table `table_Jf`.
+- The magnetic vector potential, $\vec{A}$, is computed given the current vector
+  potential computed at the first stage, $\vec{T}$. No error norms are computed
+  at this stage.
+- The magnetic vector potential computed at the preceding stage, $\vec{A}$, is
+  converted into magnetic field, $\vec{B}$. The $L^2$ norm is computed by
+  comparing the numerical result with the closed-form analytical solution. The
+  $L^2$ error norm is saved in a convergence table `table_B`.
+
+The first stage is implemented by the code contained in the name space
+`SolverT`. The third stage is implemented by the code contained in the name
+space `SolverA`. The second and the fourth stages are implemented by the code
+contained in the name space `ProjectorHcurlToHdiv`. The name space
+`ExactSolutions` contains exact closed-form analytical expressions for
+$\vec{B}$ and $\vec{J}_f$.
+
+No error norms are computed at the first and the third stages. Computing error
+norms at these stages makes no sense as we apply implicit gauges and the
+conservative portion of the solution is unknown. We can, however, judge the
+quality of the simulations at these two stages indirectly by evaluating the
+quality of the simulations at the second and the fourth stages.
```

Comment:
938 lines for the introduction makes this program effectively tied for third-longest (step-32 has 1300, step-31 has 1020, step-22 has 950). Thank you for writing so eloquently and comprehensively!